support/iconsClassesGenerator.js

```diff
@@ -24,6 +24,7 @@ const encodeSvgData = (svgData, color) => {
 	const symbols = /[\r\n%#()<>?\[\\\]^`{|}]/g;
 
 	return svgData
+		.replace(/<!--.*-->/s, '')
```

Comment:
You should use a non-greedy match here.

```
const text = '<!-- hi! -->contentz<!-- bye! -->';

const damaged = text.replace(/<!--.*-->/sg, ''); // ""

const cool = text.replace(/<!--.*?-->/sg, ''); // "contentz"
```

Also, I think you want a `g` to catch all comments.

Reply:
Well, that was close 😅